ProfessorAssessor/View Controllers/ProfessorViewController.h

```diff
@@ -1,11 +1,14 @@
 #import <UIKit/UIKit.h>
 #import "Professor.h"
+#import "Course.h"
```

Comment:
Fwiw I think you can get away with forward declarations here. In general I usually try that first and then resort to imports if it doesn't work.